### ISSUE TITLE
Why we have self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1) in Deep Q learning with Doom.ipynb

### ISSUE THREAD

**Issue by VedantYadav** [simoninithomas/Deep_reinforcement_learning_Course#1]
Hi Thomas,

I recently started working on reinforcement learning and going through your Deep reinforcement learning online series.

I don't understand that why we are doing tf.reduce_sum and multiple the convolution output to action.

```python
In [7]: ....... 
# Q is our predicted Q value
self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1)
```
Why aren't we considering self.output as predicted Q value.

**Comment by VedantYadav** [comment]
I got it.

Thanks

**Comment by 2590477658** [comment]
Hello VedantYadav, 

Sorry to bother you, do you know the meaning of tf.reduce_sum as well as the tf.multiply now? I'm confused about it.

**Comment by 2590477658** [comment]
> I got it.
> 
> Thanks

Hello VedantYadav,

Sorry to bother you, do you know the meaning of tf.reduce_sum as well as the tf.multiply now? I'm confused about it.

**Comment by MohamedJabriPharnext** [comment]
Hello

> Hi Thomas,
> 
> I recently started working on reinforcement learning and going through your Deep reinforcement learning online series.
> 
> I don't understand that why we are doing tf.reduce_sum and multiple the convolution output to action.
> 
> ```python
> In [7]: ....... 
> # Q is our predicted Q value
> self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1)
> ```
> 
> Why aren't we considering self.output as predicted Q value.

Hello, since you've managed to understand why, it will be great if you share the answer with us.

**Comment by Meur-sault** [comment]
> I got it.
> 
> Thanks

Hey, can  you explain why ?
